### DISCUSSION TITLE
Security audit

### DISCUSSION THREAD

**Discussion by enfJoao** [oqtane/oqtane.framework#4882] [General]
Hey.

One of the features I miss the most jumping in from other CMSs is the security audits / reports / checks / ...
Does Octane plan to implement some of these features? Perhaps with a custom implementation of the scheduled jobs to send the automatic reports by e-mail? 

Companies that store sensitive data would love this.

**Comment by sbwalker** [comment]
@enfJoao can you please provide more details on what you are referring to?

**Comment by enfJoao** [comment]
This:
![image](https://github.com/user-attachments/assets/ce3b3e5e-7a18-4145-b2f2-c371259b3d63)

![image](https://github.com/user-attachments/assets/5c3b59e2-d697-472e-8535-45c0abf4762d)

![image](https://github.com/user-attachments/assets/6c2b09f1-2a01-4135-b8ff-22dec6afa3bf)
